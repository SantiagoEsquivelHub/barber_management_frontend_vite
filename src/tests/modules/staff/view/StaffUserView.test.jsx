import { render } from "@testing-library/react"
import StaffUserView from "../../../../modules/staff/view/staffUser"

describe('Pruebas en <StaffUserView />', () => {

    test('Debe hacer match con el Snapshot', () => {
        const { container } = render(<StaffUserView />)
        expect(container).toMatchSnapshot();
    })

})