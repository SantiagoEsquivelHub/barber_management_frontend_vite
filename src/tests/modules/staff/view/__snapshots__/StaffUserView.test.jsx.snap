// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <StaffUserView /> Debe hacer match con el Snapshot 1`] = `
<div>
  <div
    class="contenedor_main"
  >
    Cargando...
    <div
      class="timeline_interna col-6"
    >
      <div
        class="m-3 card"
      >
        <div
          class="card-body"
        >
          <div
            class="d-flex align-items-center justify-content-between mb-2"
          >
            <div
              class="card-title h5"
            >
              Historial
            </div>
          </div>
          <div>
            Sin informaci√≥n
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
